src/hardware/serialport/softmodem.cpp

```diff
@@ -490,14 +479,16 @@ void CSerialModem::DoCommand()
 					SendRes(ResERROR);
 					return;
 				}
-				socketType = (SocketTypesE)requested_mode;
-				// This will break when there's more than two
-				// socket types.
-				LOG_MSG("SERIAL: Port %" PRIu8 " socket type %s",
-				        GetPortNumber(),
-				        socketType ? "ENet" : "TCP");
-				// Reset port state.
-				EnterIdleState();
+				if (socketType != (SocketTypesE)requested_mode) {
+					socketType = (SocketTypesE)requested_mode;
+					// This will break when there's more than two
+					// socket types.
+					LOG_MSG("SERIAL: Port %" PRIu8 " socket type %s",
+							GetPortNumber(),
+							socketType ? "ENet" : "TCP");
```

Comment:
For the two indented lines that are part of LOG_MSG: `GetPortNumber(),` and `socketType ...`

Can you tab them inward up to the same level as the `LOG_MSG`, and from there, space them inward until they line up with the first double-quote `"` in `"SERIAL`.

This way, muilti-line alignment for fuinction arguments will always be lined up, regardless of tab-size.

If you have clang-format, you can run `./scripts/format-commit.sh` or `git clang-format`, both of which will just apply formatting clean rules (like this) automatically to any changed lines in the current commit.  It's very handy for fixing stuff like this.  I think vscode and Visual Studio can auto-apply it too.